documentation/run-strategies-template.ejs

```diff
@@ -25,8 +25,10 @@ You can also [create your own](#create-your-own).
 <? }); ?>
 
 <a name="create-your-own"></a>
-#### Create your own
+#### Create Your Own
 
-You can create your own strategy by passing in a function in the `strategy` parameter to the `F.manager.RunManager()` instantiation call. 
+You can create your own strategy by:
+
+* Passing in a function in the `strategy` parameter to the `F.manager.RunManager()` instantiation call. Strategy functions must return a boolean value for whether or not to create a new run. 
```

Comment:
Not really -- strategy functions should return objects of the form

```
 {
  getRun: function(){},
  reset: function(){}
}

```

The `true/ false` only applies is the new class you create sub-classes the 'conditional creation' strategy like so

```js
var ConditionalStrategy = F.manager.RunManager.strategies.get('conditional-creation')
var myNewStrategy = new ConditionalStrategy(function() {  //return true or false here});
```